packages/types/src/index.ts

```diff
@@ -333,6 +333,13 @@ export type Leg = {
   };
   to: Place;
   transitLeg: boolean;
+  trip?: {
+    arrivalStoptime?: TripStopTime;
+    departureStoptime?: TripStopTime;
+    gtfsId?: string;
+    id: string;
+    tripHeadsign?: string;
```

Comment:
this field and `gtfsId` are both required on the schema. Should we make them required here too? Just because they're required on the schema doesn't mean we have to request them in our query, so I can see it both ways.

Reply:
The data passed on this component is sometimes generated by a client instead of fed right in from OTP. Do you think it's still worth making this change?  I guess those projects would then throw TS errors, which is good.